plugins/woocommerce/src/Blocks/BlockTypes/OrderConfirmation/Status.php

```diff
@@ -26,7 +28,7 @@ class Status extends AbstractOrderConfirmationBlock {
 	 */
 	protected function render( $attributes, $content, $block ) {
 		$order     = $this->get_order();
-		$classname = $attributes['className'] ?? '';
+		$classname = StyleAttributesUtils::get_classes_by_attributes( $attributes );
```

Comment:
Let's pass `array("extra_classes")` to be sure that we don't add unexpected classes